app/javascript/packages/countdown/countdown-alert-element.ts

```diff
@@ -0,0 +1,5 @@
+export class CountdownAlertElement extends HTMLElement {}
```

Comment:
To do: Add behavior which shows alert based on time `show-at-remaining-seconds` attribute.

Reply:
> To do: Add behavior which shows alert based on time `show-at-remaining-seconds` attribute.

Initial implementation in 63ce8e1.